### ISSUE TITLE
Fit addon not updating the height

### ISSUE THREAD

**Issue by IvanJosipovic** [BattlefieldDuck/XtermBlazor#7]
Hi,

Thank you for this awesome library!

I have been experiencing an issue where the xterm-addon-fit would not fill the full height of its parent container. I did some quick troubleshooting and it appears its due to the parent div not having a height: 100%.

Before:
![image](https://user-images.githubusercontent.com/9521987/226799539-bf058999-f33e-4760-b5a7-7c5270a70d18.png)

After:
![image](https://user-images.githubusercontent.com/9521987/226799647-dd2e9671-ad3a-4ae8-baf2-d6abbc96151e.png)

**Comment by BattlefieldDuck** [comment]
Please update the XtermBlazor to v1.8.0 and try
```razor
<Xterm Style="height:100%"/>
```

**Comment by BattlefieldDuck** [comment]
Thanks for the testing! Sadly I cannot reproduce it on my end, but I guess I know the cause of the issue.

Will release a fix soon.

**Comment by BattlefieldDuck** [comment]
> Hey,
> 
> Confirmed working, thank you!
> 
> One more note, I had to add a delay for the fit addon to work. ![image](https://user-images.githubusercontent.com/9521987/226979224-1fe82bfa-4118-4054-9553-4be630a5fc5e.png)

I just released a fix. Please try v1.8.1. I hope it fix that😄.

**Comment by IvanJosipovic** [comment]
Hey, the fix did not work for me.

**Comment by BattlefieldDuck** [comment]
Can you show the UI difference with and without `await Task.Delay(50)` ? thanks.

**Comment by IvanJosipovic** [comment]
Without Delay
![image](https://user-images.githubusercontent.com/9521987/227659593-d3c774d7-c193-4d79-922e-43948e1dac12.png)

With Delay
![image](https://user-images.githubusercontent.com/9521987/227659620-54b6f893-3d47-46a8-98d9-2cb1f019377f.png)

**Comment by BattlefieldDuck** [comment]
After viewing your KubeUI project, I discovered the cause of the problem.

The problem is it calls `await _terminal.InvokeAddonFunctionVoidAsync("xterm-addon-fit", "fit");` **before** the `ResizePanel` is resized.

You should call `await _terminal.InvokeAddonFunctionVoidAsync("xterm-addon-fit", "fit");` just after the `ResizePanel` is fully resized.

`ResizePanel`: https://github.com/IvanJosipovic/KubeUI/blob/fe2e9fa62393d38a23e473ca300e761a7be81f08/src/KubeUI.UI/Pages/Workloads/Details/Pod.razor#L40

Run `await _terminal.InvokeAddonFunctionVoidAsync("xterm-addon-fit", "fit");` somewhere else after this:
https://github.com/IvanJosipovic/KubeUI/blob/fe2e9fa62393d38a23e473ca300e761a7be81f08/src/KubeUI.UI/Components/ResizePanel.razor.cs#L52

**Comment by IvanJosipovic** [comment]
That's very interesting, thank you for looking into it. 

I would have expected that the parent components AfterRender would occur before the child.

**Comment by BattlefieldDuck** [comment]
I agree it is due to the JSInterop thing. I suggest you can `Console.WriteLine` on the C# side to see which one runs first.

Here: 
https://github.com/IvanJosipovic/KubeUI/blob/fe2e9fa62393d38a23e473ca300e761a7be81f08/src/KubeUI.UI/Components/ResizePanel.razor.cs#LL44C9-L44C9

and, here
https://github.com/IvanJosipovic/KubeUI/blob/fe2e9fa62393d38a23e473ca300e761a7be81f08/src/KubeUI.UI/Components/PodConsole.razor.cs#L64
